src/plugins/intel_cpu/tests/unit/ngraph_transformations/convert_to_leaky_relu_test.cpp

```diff
@@ -7,134 +7,134 @@
 #include <string>
 #include <memory>
 
-#include <ngraph/function.hpp>
-#include <ngraph/opsets/opset1.hpp>
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset1.hpp>
 #include <transformations/cpu_opset/common/pass/convert_to_leaky_relu.hpp>
 #include <transformations/cpu_opset/common/op/leaky_relu.hpp>
 #include <transformations/init_node_info.hpp>
 #include <transformations/utils/utils.hpp>
 #include <ov_ops/type_relaxed.hpp>
-#include <ngraph/pass/manager.hpp>
+#include <openvino/pass/manager.hpp>
 #include "common_test_utils/ngraph_test_utils.hpp"
 
 using namespace testing;
 using namespace ov::intel_cpu;
 
 TEST(TransformationTests, ConvertToLeakyReluTest1) {
-    std::shared_ptr<ngraph::Function> f(nullptr), f_ref(nullptr);
+    std::shared_ptr<ov::Model> f(nullptr), f_ref(nullptr);
```

Comment:
Very minor: we can rename `f -> m` and `f_ref -> m_ref` everywhere as well

Reply:
I will do it in the next PR